### PR TITLE
static: provide a flag to turn off community link

### PR DESCRIPTION
**Pull request by sluongng**
Provide a flag for some deployments to not display our community Slack
on the UI.

### CODE REVIEW

File: config/buildbuddy.local.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ app:
   log_error_stack_traces: true
   streaming_http_enabled: true
   invocation_log_streaming_enabled: true
+  community_link_enabed: false
```
Comment (bduffany) [inline config/buildbuddy.local.yaml:5]: revert?

File: server/static/static.go [hunk 2]
```diff
@@ -65,6 +65,7 @@ var (
 	targetFlakesUIEnabled                  = flag.Bool("app.target_flakes_ui_enabled", false, "If set, show some fancy new features for analyzing flakes.")
 	codeEditorV2Enabled                    = flag.Bool("app.code_editor_v2_enabled", false, "If set, show v2 of code editor that stores state on server instead of local storage.")
 	bazelButtonsEnabled                    = flag.Bool("app.bazel_buttons_enabled", false, "If set, show remote bazel buttons in the UI.")
+	communityLinkEnabled                   = flag.Bool("app.community_link_enabed", true, "If set, show a link to BuildBuddy community in the UI.")
```
Comment (bduffany) [inline server/static/static.go:68]:
```suggestion
	communityLinksEnabled                   = flag.Bool("app.community_links_enabed", true, "If set, show a link to BuildBuddy community in the UI.")
```

File: proto/config.proto [hunk 3]
```diff
@@ -174,6 +174,9 @@ message FrontendConfig {
 
   // The Content Security Policy nonce to use for inline <style> elements.
   string csp_nonce = 57;
+
+  // Whether the community link is enabled in the UI.
+  bool commmunity_link_enabled = 58;
```
Comment (bduffany) [inline proto/config.proto:179]: `mmm` => `mm`

also `link` => `links` since this enables all links

```suggestion
  bool community_links_enabled = 58;
```

**Review by bduffany — APPROVED** [review]

File: config/buildbuddy.local.yaml (hunk 1 above)
Comment (sluongng) [reply config/buildbuddy.local.yaml]: ah damn I forgot to revert after my local test

File: server/static/static.go (hunk 2 above)
Comment (sluongng) [reply server/static/static.go]: good call. Also changed the flag doc.

File: proto/config.proto (hunk 3 above)
Comment (sluongng) [reply proto/config.proto]: Dyslexia 😵‍💫
Fixed.

**Pull request merged**